resources/views/dashboard/admin/realops/index.blade.php

```diff
@@ -26,99 +26,103 @@
 @endif
 
 </div>
-<table class="table table-bordered table-striped text-center">
```

Comment:
I'm having trouble following the changes that you've made to this table (the flight listing). Is this change in-scope for the current PR or is it related to something else? It looks like there are key pieces of code/capabilities that are missing/have been removed and I can't understand why.

Reply:
The only change here should be adding the if around it so it'll show the table if flights exist, and the empty state if not. I think some GitHub diff weirdness is making it look like things in the table actually changed... look at the commit without the diff and it should be more clear that only the if was added

Reply:
ok, strange - I will take a closer look!